tests/envs/test_pybullet_blocks.py

```diff
@@ -40,10 +41,22 @@ def PutOnTable(self):
         return self._PutOnTable
 
     def set_state(self, state):
```

Comment:
is this method not implemented in the actual env? why do we not need it there?

Reply:
it's not, we don't necessarily wanna assume that we can set the state in the actual env because it's a proxy for the real world